### PR TITLE
A minor change to add VirtualBox in 'PRE-REQS / SCRIPT EXPECTATIONS'

### PR DESCRIPTION
**Pull request by cgx027**
A minor change to add VirtualBox in 'PRE-REQS / SCRIPT EXPECTATIONS' section.

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ to connect DHCP and TFTP from RackHD to the PXE clients.
 ## PRE-REQS / SCRIPT EXPECTATIONS
 
 We expect the latest version of git, [Vagrant](https://www.vagrantup.com/downloads.html),
+[VrtualBox](https://www.virtualbox.org/wiki/Linux_Downloads),
```
Comment (heckj) [inline example/README.md]: Could you change the link to generically virtualbox.org instead of a platform specific download?

Comment (cgx027) [inline example/README.md]: Make sense. Just update the link with a new commit.

File: example/README.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ to connect DHCP and TFTP from RackHD to the PXE clients.
 ## PRE-REQS / SCRIPT EXPECTATIONS
 
 We expect the latest version of git, [Vagrant](https://www.vagrantup.com/downloads.html),
+[VrtualBox](https://www.virtualbox.org/wiki/Downloads),
```
Comment (yyscamper) [inline example/README.md]: typo: should be "VirtualBox"

**Comment by heckj** [comment]
+1 with the typo fixed

**Comment by benbp** [comment]
:+1: with the typo fixed as well.

**Comment by cgx027** [comment]
Thanks guys. Can't believe myself on the Typo! Done fixing it.

**Comment by benbp** [comment]
Do we also need to specify installing the VirtualBox extensions pack?

**Pull request merged**